app/models/mongoidable/policy_relation.rb

```diff
@@ -2,30 +2,34 @@
 
 module Mongoidable
   class PolicyRelation
+    extend Memoist
     include Mongoid::Document
     include Mongoidable::Document
 
     field :requirements, type: Hash
     belongs_to :policy, class_name: Mongoidable.configuration.policy_class, polymorphic: true
 
-    def add_inherited_abilities
-      @abilities.merge(merge_requirements)
+    def inherited_abilities
+      merged_policy_requirements
     end
 
-    def merge_requirements
-      result = Mongoidable::Abilities.new(mongoidable_identity, self)
-      return result unless policy
+    def merged_policy_requirements
+      merged_policy_requirements = Mongoidable::Abilities.new(mongoidable_identity, self)
```

Comment:
This is a comment on what may just be a personal foible.

I don't like creating a local variable anywhere in the code that is the same as a method name (even in that method.)

I understand why you would do it, but I find it confusing in the code to look at a single line and have to think about/wonder:  Is this a method call or a variable?